apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -110,6 +127,20 @@ const InnerFileBrowser = React.memo(
                   <span onClick={() => deleteFile(f.id)}>
                     <i className="fa-solid fa-trash" />
                   </span>
+                  {isStartMode && (
+                    <span
+                      onClick={e => {
+                        e.stopPropagation();
```

Comment:
why do we need `e.stopPropagation` here but not for other on clicks in this file?

Reply:
Hmm, we don't...